website/content/en/status/report-2025-10-2025-12/accessibility-handbook.adoc

```diff
@@ -0,0 +1,32 @@
+=== Accessibility Handbook
+
+Link: +
+link:https://docs.freebsd.org/en/books/accessibility/[FreeBSD Accessibility Handbook] URL: link:https://docs.freebsd.org/en/books/accessibility/[]
+
+Contact: FreeBSD Accessibility mailing list <freebsd-accessibility@FreeBSD.org> +
+Contact: Alfonso Sabato Siciliano <asiciliano@FreeBSD.org>
+
+In recent reports, the new Accessibility Handbook was announced and its reviews were published.
+The handbook aims to document the assistive technologies available in FreeBSD, covering both the base system and the Ports Collection.
+
+During this quarter:
+
+* The book was presented at the FreeBSD Developer Summit Zagreb 2025, with slides available at link:https://docs.google.com/presentation/d/1rnw5fU8KyUCGNqR6gkM5IBdyx8cVVdqigCGmrrd--a4[slides].
```

Comment:
Unless you want the sentence to be like this for accessibility reasons, I suggest the following:

```suggestion
* The book was presented at the FreeBSD Developer Summit Zagreb 2025, with slides available link:https://docs.google.com/presentation/d/1rnw5fU8KyUCGNqR6gkM5IBdyx8cVVdqigCGmrrd--a4[here].
```